tests/playwright/src/specs/podman-machine-rootless.spec.ts

```diff
@@ -0,0 +1,76 @@
+/**********************************************************************
+ * Copyright (C) 2024 Red Hat, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ ***********************************************************************/
+import * as os from 'node:os';
+
+import type { Page } from '@playwright/test';
+import { expect as playExpect } from '@playwright/test';
+import { afterAll, beforeAll, beforeEach, describe, test } from 'vitest';
+
+import { CreateMachinePage } from '../model/pages/create-machine-page';
+import { ResourcesPage } from '../model/pages/resources-page';
+import { ResourcesPodmanConnections } from '../model/pages/resources-podman-connections-page';
+import { WelcomePage } from '../model/pages/welcome-page';
+import { NavigationBar } from '../model/workbench/navigation';
+import { PodmanDesktopRunner } from '../runner/podman-desktop-runner';
+import type { RunnerTestContext } from '../testContext/runner-test-context';
+import { deletePodmanMachine } from '../utility/operations';
+
+let pdRunner: PodmanDesktopRunner;
+let page: Page;
+let navBar: NavigationBar;
+const PODMAN_MACHINE_NAME: string = 'Podman Machine Rootless';
+
+beforeAll(async () => {
+  pdRunner = new PodmanDesktopRunner();
+  page = await pdRunner.start();
+  pdRunner.setVideoAndTraceName('podman-rootless-machine-e2e');
+
+  const welcomePage = new WelcomePage(page);
+  await welcomePage.handleWelcomePage(true);
+  navBar = new NavigationBar(page);
+});
+
+afterAll(async () => {
+  await deletePodmanMachine(page, PODMAN_MACHINE_NAME);
```

Comment:
I am truly curious how this test is going to work with a test env. in mind. Like we have usually podman machine already present. This means that the connection could be switched. But at the test end, we need to make sure switch it back again. 
This test would need some parameters to operate the workflow in a right way. and to fix the test among the rest.